### ISSUE TITLE
Enable limit, orderby for delete

### ISSUE THREAD

**Issue by jackghg** [sqlitebrowser/sqlitebrowser#2739]
Limit with delete doesnt work. It needs this flag that should be enabled.
https://www.sqlite.org/compile.html#enable_update_delete_limit

**Comment by justinclift** [comment]
Interesting.  Didn't even know that's a thing. :wink:

Well, we could enable it in our development builds and see if it has any bad effects.

@jackghg Hmmm, if you're running it on Arch, then it's probably the SQLite package on Arch that needs it.  We'll just use whichever SQLite "the OS provides". :smile:

**Comment by chrisjlocke** [comment]
> Interesting. Didn't even know that's a thing. 😉

Me neither! 😁😂

**Comment by jackghg** [comment]
Thanks. Well the delete could be some nano-seconds slower I guess...

**Comment by justinclift** [comment]
@jackghg Yeah, no idea yet. :wink:

Which OS are you using our application on?  Asking because I'd be interested hearing how it goes with that too. :smile:

**Comment by jackghg** [comment]
Artix btw. But i dont know if i have the time to compile it to test... sorry

**Comment by justinclift** [comment]
No worries.  I guess I was more meaning that you'll probably need to file an issue with Artix or Arch, to get the option enabled in it's SQLite package there.

With that in place, you should (in theory) automatically have it available in your DB Browser for SQLite. :wink:

**Comment by justinclift** [comment]
Seems like @AndyRandom might be the person to ask? :smile:

**Comment by jackghg** [comment]
I wonder if on linux it uses the system libsqlite or its own (like a standalone app) ?

**Comment by justinclift** [comment]
Yeah, it's a good question.  Do you remember if you used the Arch package, or was it maybe a Snap, or AppImage? :smile:

If it was the Artix supplied package (which seems to be Arch packages?), then it'll probably use the system SQLite library.  With the Snap and AppImage, they probably include their own. :smile:

**Comment by scottfurry** [comment]
Artix is similar to Arch with the exception of systemd.
Both use latest/greatest/rolling release.
Currently, system sqlite library is at 3.35.5 (shown [here](https://gitea.artixlinux.org/packagesS/sqlite) )
